src/Calculator/Common/KeyboardShortcuManager.cs

```diff
@@ -714,6 +717,29 @@ private static void OnKeyDownHandler(CoreWindow sender, KeyEventArgs args)
                 }
             }
 
+            private static void OnKeyUpHandler(CoreWindow sender, KeyEventArgs args)
+            {
+                s_keyHandlerCount--;
+                if (s_keyHandlerCount == 0 && s_deferredEnableShortcut)
+                {
+                    KeyboardShortcutManager.DisableShortcuts(false);//shortcuts enabled
+                    s_deferredEnableShortcut = false;
+                }             
+
+            }
+
+            public static void TryEnableShortcut()
+            {
+                if (s_keyHandlerCount > 0)
+                {
+                    s_deferredEnableShortcut = true;
+                }
+                else
+                {
+                    KeyboardShortcutManager.DisableShortcuts(true);
```

Comment:
This should be passing `false`. In both cases, we should be passing the same value and getting the same behavior - the only difference is whether it happens immediately in `TryEnableShortcut` or later in `OnKeyUpHandler`.